common/db.go

```diff
@@ -0,0 +1,109 @@
+package common
+
+import (
+	"bytes"
+	"database/sql"
+	"errors"
+	"math/big"
+	"text/template"
+
+	"github.com/golang/glog"
+	_ "github.com/mattn/go-sqlite3"
+)
+
+type DB struct {
+	dbh *sql.DB
+
+	// prepared statements
+	updateKV *sql.Stmt
+}
+
+var LivepeerDBVersion = 1
+
+var ErrDBTooNew = errors.New("DB Too New")
+
+var schema = `
+	CREATE TABLE IF NOT EXISTS kv (
+		key STRING PRIMARY KEY,
+		value STRING,
+		updatedAt STRING DEFAULT CURRENT_TIMESTAMP
+	);
+	INSERT OR IGNORE INTO kv(key, value) VALUES('dbVersion', '{{ . }}');
+	INSERT OR IGNORE INTO kv(key, value) VALUES('lastBlock', '0');
```

Comment:
So for more complex structures like a job, we'll just serialize/deserialize?  What if we wanted to do a query based on an attribute?

Reply:
nvm.  Doing review for the jobs PR.